src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/analysis/AnalysisDetailsPageIT.java

```diff
@@ -40,7 +42,7 @@ public void testPageSetUp() throws URISyntaxException, IOException {
 		// Ensure the input files are displayed
 		// 2 Files expected since they are a pair.
 		page.displayInputFilesTab();
-		assertEquals("Should display 1 pair of paired end files", 2, page.getNumberOfPairedEndInputFiles());
+		assertEquals("Should display 1 pair of paired end files", 1, page.getNumberOfPairedEndInputFiles());
```

Comment:
Could you add a test case where there is no sample linked to the sequencing object, and yet you still get the sequence files displayed on the page?